src/Control/Logger/Katip.hs

```diff
@@ -14,12 +14,11 @@ import           Control.Has.Katip ()
 import           Control.Lens hiding ((.=))
 import           Control.Logger.Internal
 import           Data.Aeson hiding (Error)
-import qualified Data.HashMap.Strict as HM
 import           Data.Text (Text)
 import qualified Data.Text as T
 import qualified Data.Text.Lazy.Builder as T.Builder
 import           Katip as K hiding (logMsg)
-import           Katip.Core (ItemFunc(..), LocJs(..), ProcessIDJs(..))
```

Comment:
`ItemFunc` comes from the custom katip, so I had to rework the related code not to use it.